tests/tests/callbacks.rs

```diff
@@ -216,3 +216,92 @@ mod return_result_skip {
         assert_eq!(lexer.next(), Some(Err(LexerError::UnterminatedComment)));
     }
 }
+
+mod skip_callback_function {
+    use super::*;
+
+    #[derive(Debug, Default, PartialEq, Clone)]
+    enum LexerError {
+        #[default]
+        Other,
+    }
+
+    #[derive(Logos, Debug, PartialEq)]
+    #[logos(skip r"[ \t\n\f]+")]
+    #[logos(skip("<!--", skip_comment))]
+    #[logos(error = LexerError)]
+    enum Token<'src> {
+        #[regex(r"<[a-zA-Z0-9-]+>", |lex| &lex.slice()[1..lex.slice().len()-1])]
+        Tag(&'src str),
+    }
+
+    fn skip_comment<'src>(lexer: &mut Lexer<'src, Token<'src>>) {
+        let end = lexer
+            .remainder()
+            .find("-->")
+            .unwrap_or(lexer.remainder().len());
```

Comment:
In line with other comment, you can probably fix the text by changing this to `.map(|id| id + 3).unwrap_or(lexer.remainer().len());` and `bump(end)`.

Reply:
Thanks!